llvm-core/clang/clang-19.1.6.ebuild

```diff
@@ -260,6 +260,7 @@ multilib_src_configure() {
 		-DCMAKE_INSTALL_MANDIR="${EPREFIX}/usr/lib/llvm/${LLVM_MAJOR}/share/man"
 		-DLLVM_ROOT="${EPREFIX}/usr/lib/llvm/${LLVM_MAJOR}"
 		-DCLANG_CONFIG_FILE_SYSTEM_DIR="${EPREFIX}/etc/clang"
+		-DCLANG_CONFIG_FILE_USER_DIR="~/.config/clang"
```

Comment:
Oh, does it work now?

Ohh, I see:

```
commit b3f94fe1c3c4f4fd7a937288b5543c4067b3e2f2
Author:     Michał Górny <mgorny@gentoo.org>
AuthorDate: 2022-11-16 13:20:02 +0100
Commit:     Michał Górny <mgorny@gentoo.org>
CommitDate: 2022-11-16 13:23:25 +0100

    [clang][Driver] allow tilde in user config dir
    
    This patch allows users to configure clang with option
    e.g. `-DCLANG_CONFIG_FILE_USER_DIR=~/.config/clang` or invoke clang
    with `--config-user-dir=~/.config/clang`.
    
    Patch merged on behalf of @paperchalice (LJC)
    
    Differential Revision: https://reviews.llvm.org/D136940
```

Forgot about it xD.

Reply:
I didn't realise that was you! Glad I picked the same directory as your example too.

Reply:
Not me, I just committed it on behalf of the person.